src/app/components/JumpTo/index.tsx

```diff
@@ -25,9 +23,6 @@ const JumpTo = ({ jumpToHeadings, showRelatedContentLink }: JumpToProps) => {
     optimizely,
   };
 
-  // TODO: Remove for release
-  if (isLive()) return null;
```

Comment:
This may be a silly question (and not exactly related to this PR 😅 ) but... how does the experiment limit the Jump To component to the Hindi service only? Is that an Optimizely thing? I had a thought that some services might not want this component, and therefore we should probably hide it behind a toggle. Or this that a question for later, once the experiment is complete, and we know that the future of this component will be?

Reply:
Yea Optimizely has all of our services in as an 'audience', so it will only run and show the component if people meet the conditions of the experiment, so 'hindi'.

The component display logic is: https://github.com/bbc/simorgh/blob/aeac98ab69d3b80de0c8c717f71a1561b13be6d5/src/app/pages/ArticlePage/ArticlePage.tsx#L242-L245

So will only ever show when the Optimizely variation is `on`

Reply:
Thanks for the explanation, I didn't know how Optimizely was configured, so this makes total sense!

Reply:
I don't know why we could see it on Pidgin though 🫨